simpleclient/src/main/java/io/prometheus/client/SimpleCollector.java

```diff
@@ -209,6 +215,13 @@ public B namespace(String namespace) {
       this.namespace = namespace;
       return (B)this;
     }
+    /**
+     * Set the unit of the metric. Required.
```

Comment:
Isn't a unit optional, not required? From the [OpenMetrics spec](https://github.com/OpenObservability/OpenMetrics/blob/2474ddafe93217cc4979de0b3148e47a1d9340ad/specification/OpenMetrics.md#unit):

> If non-empty, it MUST be a suffix of the MetricFamily name separated by an underscore.

Reply:
Indeed this is a copy&paste error, would you like to send a PR to fix this?

Reply:
Sure. I've opened https://github.com/prometheus/client_java/pull/619